docs/_static/site.css

```diff
@@ -23,3 +23,28 @@
 	height: 3rem;
 }
 
+:root {
+	--primary-accent: #4ba3d9;
+	--base-color: #f7f7f7;
+	--iver-yelow: #fcde06;
+	--mono-hue: 218;
+	--mono-saturation : 12%;
+	--base-background-color: #292c31;
+	--sidebar-name-color: var(--primary-accent);
+	--sidebar-nav-link-color--active: var(--primary-accent);
+	--code-theme-keyword: var(--primary-accent);
+	--sidebar-background: #212122;
+	--mark-background:  var(--iver-yelow);
+	--code-theme-variable: var(--iver-yelow);
+	--code-theme-punctuation: var(--base-color);
+	--code-theme-text: var(--base-color);
+	--code-theme-selector: var(--iver-yelow)
+}
+
+.app-name-link {
+	font-size: 37px;
+}
+
+.sidebar-nav p.active > a {
+	color: #4ba3d9;
+}
```

Comment:
Sorry, but the coverpage seems a bit oddly colored.

- Could you make the Wharf icon white or blue? 
- The subtitle being yellow really makes it stand out, but kind of in a bad way as pretty much nothing else is yellow and IMO the contrast scars a little.
- Is it possible to make the "Documentation" button stand out more? In the Adobe XD document they actually used blue text and border for buttons, maybe that will be suitable here as well?

Reply:
Sure, white vs blue logo:

![image](https://user-images.githubusercontent.com/22037089/111812324-4d87f000-88d8-11eb-9854-4750430244dc.png)

![image](https://user-images.githubusercontent.com/22037089/111812433-6c868200-88d8-11eb-842a-d80e30802688.png)

Documentation button variants:

1:
![image](https://user-images.githubusercontent.com/22037089/111812830-c38c5700-88d8-11eb-9333-9a4133d182ef.png)

2:
![image](https://user-images.githubusercontent.com/22037089/111812934-e74f9d00-88d8-11eb-8cfa-f3604d9b9cf5.png)

3:
![image](https://user-images.githubusercontent.com/22037089/111813083-0817f280-88d9-11eb-867a-7ab575b54708.png)

Reply:
Looks nice! I vote for the white logo.

Reply:
blue 3

White icon made it pop too much, but blue icon made eyes focus on the text, which is good

Really like the inverse effect in the buttons on example 3. Also, that's what we have in the adobe XD design
![image](https://user-images.githubusercontent.com/2477952/111814238-8759f600-88da-11eb-9b77-aebd7af750e7.png)

Reply:
I also like blue3, which is actually what I pushed pre-emptively ;)

Ok to resolve @martin-linden? I agree with @jilleJr that the white logo stands out too much.

Reply:
Or if we want the icon to pop then we go with white

Lol spending way too much time on colors, this is amazing :D First time for anything, right?

Ok I'm agreeing with @martin-linden here on white icon. I think.

Reply:
Sure! Blue looks nice too. Actually kind of like it more now when I looked at it again.

Reply:
Hmm GitHub doesnt live update discussions. Buu

I'm revoking my votes. Go with what feels best

Reply:
Personally, I think the logo is a bit too big. If we want the text to be the focus here I think we should increase the size of the logo at least a little bit.

Reply:
Actually, nevermind. It's probably just because the image is zoomed in on the area here that it appears so big.

Reply:
Sounds like I can resolve this. Here's how it looks now:

![image](https://user-images.githubusercontent.com/22037089/111822753-7ada9b00-88e4-11eb-9586-a997a4f65d80.png)

Reply:
looks great :)